README.md

```diff
@@ -1,16 +1,46 @@
 # ALKiln
-Integrated automated end-to-end testing with docassemble, puppeteer, cucumber, and chai.
 
-Works well with https://github.com/suffolklitlab/docassemble-AssemblyLine but isn't
-dependent on it.
+ALKiln, or Assembly Line Kiln, is an open source project that helps [docassemble](https://docassemble.org/) developers create automated tests for their interviews (online forms). It works well with https://github.com/suffolklitlab/docassemble-AssemblyLine but isn't dependent on it.
```

Comment:
```suggestion
ALKiln, or Assembly Line Kiln, is an open source project that helps [docassemble](https://docassemble.org/) developers create automated tests for their interviews (online forms). It works well with the [Document Assembly Line Project](suffolklitlab.org/docassemble-AssemblyLine-documentation) but isn't dependent on it.
```

Reply:
Maybe something more narrative if we can think of it at some point. Like [ALWeaver's README](https://github.com/SuffolkLITLab/docassemble-ALWeaver#readme)'s first paragraph.